core/common/src/main/java/alluxio/conf/InstancedConfiguration.java

```diff
@@ -86,7 +84,11 @@ public String get(PropertyKey key, ConfigurationValueOptions options) {
       throw new RuntimeException(ExceptionMessage.UNDEFINED_CONFIGURATION_KEY.getMessage(key));
     }
     if (!options.shouldUseRawValue()) {
-      value = lookup(value);
+      try {
+        value = lookup(value);
+      } catch (UnresolvablePropertyException e) {
+        throw new RuntimeException(e);
```

Comment:
it would be handy to include the `key` value that couldn't be resolved here

Reply:
oh right, forgot that it could propagate the sub-keys. It could still be handy though to tell the user the original key that was requested in the failed `get`